union-ai-admin/aws/unionai-provisioner-role.template.yaml

```diff
@@ -60,12 +60,6 @@ Resources:
             Resource:
               - arn:aws:s3:::union-cloud-*
               - arn:aws:s3:::union-cloud-*/*
-          - Action:
-              - cloudfront:CreateCloudFrontOriginAccessIdentity
-              - cloudfront:DeleteCloudFrontOriginAccessIdentity
```

Comment:
@evalsocket we'll need to leave the Create and Delete permissions around for now, as we still need to apply a change to actually delete these before we can remove these permissions.